nuitka/nodes/BuiltinAbsNodes.py

```diff
@@ -0,0 +1,41 @@
+#     Copyright 2019, Kay Hayen, mailto:kay.hayen@gmail.com
+#
+#     Part of "Nuitka", an optimizing Python compiler that is compatible and
+#     integrates with CPython, but also works on its own.
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+#
+""" Built-in abs nodes. """
+
+from nuitka.specs import BuiltinParameterSpecs
+
+from .ExpressionBases import ExpressionBuiltinSingleArgBase
+
+
+class ExpressionBuiltinAbs(ExpressionBuiltinSingleArgBase):
+    kind = "EXPRESSION_BUILTIN_ABS"
```

Comment:
I would have expected this one in `OperatorNodes`. There is going to be a lot of commonality with other unary operations. `|a|` in math is an unary operation, just not one for which there is Python syntax, i.e. the built-in is the only way to say it. That doesn't mean it shouldn't be where the others are.